Adapt documentation to new split binaries from C++ rewrite. (#2289)
mendersoftware/mender-docs: 06.Artifact-creation/06.Standalone-deployment/docs.md

## Setting Mender up for standalone mode

If you would like to run Mender in *standalone* mode, you

> must make sure that the Mender-update client does *not run as a daemon*. In most cases this
> will entail disabling or removing any `systemd` unit that starts the Mender-update client. If you want to check if Mender is running as a daemon, you can try the following command:

Reviewer: ```suggestion
must make sure that the `mender-updated` service isn't running. If you want to check if Mender is running as a daemon, you can try the following command:
```
